conductor/__beta__/job/worker.py

```diff
@@ -16,9 +16,14 @@ class WorkerJob(job.Job):
 class DeadlineWorkerJobError(WorkerJobError):
     pass
 
+class DeadlineToConductorPackageMapperError(Exception):

```

Comment:
Nit: Should inherit from a `ConductorError` class, that inherits from `Exception` itself.

Reply:
Agree - I was going to do that once I migrate all this to `conductor-core`. Would you prefer I do it now?